app/cafilters.go

```diff
@@ -0,0 +1,90 @@
+package app
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/temporalio/tcld/api/temporalcloudapi/namespace/v1"
+)
+
+type certificateFilter struct {
+	CommonName             string `json:"commonName"`
+	Organization           string `json:"organization"`
+	OrganizationalUnit     string `json:"organizationalUnit"`
+	SubjectAlternativeName string `json:"subjectAlternativeName"`
+}
+
+type certificateFiltersConfig struct {
```

Comment:
open to having this just be a top-level array vs an array embedded in a field. thoughts here?

Reply:
I think this is better. Gives us an option to put more information in the object if the need ever arises.